packages/auto-pr-labeler/src/assignLabels.ts

```diff
@@ -1,12 +1,56 @@
 import * as core from '@actions/core';
 
+import { espressoStaff } from './espressoStaff';
 import { PR_REVIEW_DECISION, PR_STATE } from './constants';
 import { repoLabels } from './labels';
-import { addLabelsMutation, removeLabelsMutation } from './mutations';
+import { addAssigneesMutation, addLabelsMutation, removeLabelsMutation } from './mutations';
 
-import type { ID, IssueConnection, LabelList, LabelsQueryResponse, PullRequest, RepoName } from './types';
+import type {
+	AssigneesQueryResponse,
+	ID,
+	IssueConnection,
+	LabelList,
+	LabelsQueryResponse,
+	PullRequest,
+	RepoName,
+} from './types';
 import type { GraphQlQueryResponse } from '@octokit/graphql/dist-types/types';
 
+/**
+ * adds the supplied assignees to the specified Issue or Pull Request
+ *
+ * @param assignableId ID
+ * @param assigneeIds [ID]
+ * @returns Promise<GraphQlQueryResponse<AssigneesQueryResponse>>
+ */
+const addAssignees = async (
+	assignableId: ID,
+	assigneeIds: Array<ID>
+): Promise<GraphQlQueryResponse<AssigneesQueryResponse>> => {
+	try {
+		// eslint-disable-next-line no-console
+		console.log('%c addAssignees', 'color: HotPink;', { assignableId, assigneeIds });
+		return await addAssigneesMutation(assignableId, assigneeIds);
+	} catch (error) {
+		core.setFailed(error.message);
+	}
+};
+
+/**
+ * assigns Q/A staff to the specified Pull Request
+ *
+ * @param assignableId ID
+ * @param assigneeIds [ID]
```

Comment:
This parameter does not exist in the actual function signature?

Reply:
good eye, thnx,  will fix